src/components/Superfest/QuestCard/QuestCard.tsx

```diff
@@ -55,32 +56,34 @@ export const QuestCard = ({
 
   return (
     <QuestCardMainBox>
-      <Box sx={{ display: 'flex', flexDirection: 'row' }}>
-        {image && (
-          <Image
-            src={image}
-            alt="Quest Card Image"
-            width={256}
-            height={256}
-            style={{
-              borderTopLeftRadius: '8px',
-              borderTopRightRadius: '8px',
+      <Link href={`/superfest/${slug}`}>
+        <Box sx={{ display: 'flex', flexDirection: 'row' }}>
+          {image && (
+            <Image
+              src={image}
+              alt="Quest Card Image"
+              width={288}
+              height={288}
+              style={{
+                borderTopLeftRadius: '8px',
+                borderTopRightRadius: '8px',
+              }}
+            />
+          )}
+          <Box
+            sx={{
+              position: 'relative',
+              marginLeft: '-32px',
+              maringTop: '-16px',
             }}
-          />
-        )}
-        <Box
-          sx={{
-            position: 'relative',
-            marginLeft: '-32px',
-            maringTop: '-16px',
-          }}
-        >
-          {rewards?.amount && <OPBadge />}
+          >
```

Comment:
Since you´re cleaning up so nicely, maybe this could also become it´s own styled-component?